### ISSUE TITLE
Add the ability to filter test cases (multiselection for custom fields)

### ISSUE THREAD

**Issue by Nadia0902** [testomatio/app#1120]
потрібно, щоб кожен тест-кейс можна було позначати кількома значеннями, а потім мати можливість створювати тест-ран із різним набором фільтрів.
1. Є можливість створити параметр і до нього значення. https://share.cleanshot.com/Vtvj3vH5
2. Цей параметр можна додати до всіх тест кейсів https://share.cleanshot.com/2qxvkc5X
3. При створенні тесту є можливість вибору кількох значень до тесту https://share.cleanshot.com/Ms8Q2xWp
4. При сторенні рану для регрессії є можливість обрати по фільтрам тести, які будуть додані для проходження рану https://share.cleanshot.com/C69gSNGz

Це вирішує проблему сортування кейсів по платформі або мікросервісам. 
В нас мікросервіна архітектура і багато платформ на яких потрібно проводити тесутвання. Щоб проводити регерссію на різних платформах і точково обирати якого мікросервісу стосується функціонал ми викорисовуємо фільтрацію, це допомагає зменшити кількість тестів і більш якісно і точково провести тестування сервісу, який збираємось релізити.

**Comment by poliarush** [comment]
@littlefrontender do you need backend here?

**Comment by littlefrontender** [comment]
@poliarush yes, it requires backend. @DavertMik mentioned that he would consider it.

**Comment by TetianaKhomenko** [comment]
Hi @Nadia0902! The feature is live now https://changelog.testomat.io/new-features-for-custom-fields-ai-generated-bdd-scenarios-enhanced-exports-and-more/

**Comment by Nadia0902** [comment]
Cool, thanks! I'll give it a go.

**Comment by iaza18** [comment]
PR: https://github.com/testomatio/docs/pull/255

**Comment by iaza18** [comment]
PR: https://github.com/testomatio/docs/pull/273

**Comment by iaza18** [comment]
https://docs.testomat.io/advanced/tags-labels/
